cmake/EmbedLLVM.cmake

```diff
@@ -0,0 +1,40 @@
+enable_language(ASM)
+
+function(EMBED_LLVM OUTPUT_FILE)
+    cmake_parse_arguments(llvm "" "EXPORT_MACRO" "" ${ARGN})
+    set(INPUT_FILES "${llvm_UNPARSED_ARGUMENTS}")
+
+    set(CPP_BUNDLE_FILE "${CMAKE_CURRENT_BINARY_DIR}/embed_llvm/jit_llvm_bundle.cpp")
```

Comment:
This files contains a number of references to files and functions in our code. Could you add some short comments on what they are good for?

Reply:
I am not sure I understand what you mean... `embed_llvm/jit_llvm_bundle.cpp` is compile-time generated to include all the files listed [here](https://github.com/hyrise/hyrise/pull/696/files#diff-c48b16f2a0840b5b7dbcb9e27816ecbaR353). The file is then compiled to LLVM make all of the code from those files available to the Jit at runtime

Reply:
That is probably obvious to you, but for me, the relation between those two was not clear. I am a bit worried that this file is hard to understand without comments.